src/Organizations/OrganizationForm/OrganizationContactPeopleForm/OrganizationContactPeopleList.js

```diff
@@ -87,10 +91,12 @@ const alignRowProps = { alignLastColToEnd: true };
 
 const OrganizationContactPeopleList = ({ fetchContacts, fields, contactsMap, orgId, categoriesDict, stripes }) => {
   const intl = useIntl();
-  const contentData = fields.getAll().map((contactId, _index) => ({
-    ...get(contactsMap, contactId, {}),
-    _index,
-  }));
+  const contentData = useMemo(() => (
+    sortBy(fields.getAll().map((contactId, _index) => ({
+      ...get(contactsMap, contactId, {}),
+      _index,
+    })), 'firstName')),
+  [fields, contactsMap]);
```

Comment:
contactsMap is re-created on every render

Reply:
the same for `fields`.